climada/hazard/plot.py

```diff
@@ -65,6 +67,37 @@ def plot_rp_intensity(self, return_periods=(25, 50, 100, 250),
                                         colbar_name, title, smooth=smooth, axes=axis,
                                         figsize=figsize, adapt_fontsize=adapt_fontsize, **kwargs)
         return axis, inten_stats
+    
+    def plot_local_rp(self, threshold_intensities, smooth=True, axis=None, figsize=(9, 13), adapt_fontsize=True, cmap = 'viridis_r', **kwargs):
```

Comment:
So, if a user wants to adapt the say color map, they must recompute the return periods every time? This seems rather inefficient. Would it not make sense to be able to pass the return periods to this function? Something like

```
plot_local_rp(self, threshold_intensities, return_periods=None, smooth=True, axis=None, figsize=(9, 13), adapt_fontsize=True, cmap = 'viridis_r', **kwargs):
```

Then, if return_periods is none, compute them. Else, use the return periods to make the graph. Maybe throw a warning if the threshold_itnensities are not in the gdf.

Reply:
sorry, I forgot to remove this function. The functionality is now in `subplots_from_gdf`